components/automate-ui/src/app/components/input/input.directive.scss

```diff
@@ -19,7 +19,8 @@
   width: 100%;
   background-color: $white;
   padding: 1em;
-  font-size: 0.75em;
+  // testing
+  // font-size: 0.75em;
```

Comment:
@susanev - Is there a uniform font size we want to be using across all inputs?  I found that different inputs of the same type (i.e. all type="text" inputs) were in certain cases, using a very slightly different font size.  Removing the font size here set them all to uniform, but I'm thinking it is probably inheriting whatever our 1em is set to.

Reply:
14px would be so rad!